manifests/windows.pp

```diff
@@ -84,7 +84,7 @@
         inherit_parent_permissions => false,
         permissions                => [
           { 'identity' => 'NT AUTHORITY\SYSTEM', 'rights' => ['full'] },
-          { 'identity' => 'BUILTIN\Administrators', 'rights' => ['full'] },
+          { 'identity' => 'S-1-5-32-544', 'rights' => ['full'] },
```

Comment:
maybe add a comment here, what `S-1-5-32-544` is, where it comes from? so that one can know without parsing git history/merges